components/tensorboard-controller/Makefile

```diff
@@ -40,9 +39,16 @@ help: ## Display this help.
 
 ##@ Development
 
+<<<<<<< HEAD
```

Comment:
Some conflict resolution needs to happen in this file as well

Reply:
Done!